julia/src/intermediate2daily.jl

```diff
@@ -42,7 +50,12 @@ try
       trajectory_group["daily_detections"] = daily( detection_times |> trajectory, max_days)
       trajectory_group["daily_deaths"] = daily(infection_times.+death_progressions |> trajectory, max_days)
       trajectory_group["daily_hospitalizations"] = daily(infection_times.+hospitalization_progressions |> trajectory, max_days)
-  
+      for i in values(instances(ContactKind))
+        if i == NoContact
+          continue
+        end
+        trajectory_group["daily_" * lowercase(string(i))] = daily(infection_times, contact_kinds, i |> UInt8, max_days)
+      end
```

Comment:
This should do the same thing
```suggestion
      for kind in values(ContactKind)
        if kind == NoContact
          continue
        end
        trajectory_group["daily_" * lowercase(string(i))] = daily(infection_times[contact_kinds.==Int(kind)], max_days)
      end
```

Reply:
sounds good

Reply:
without using `instances` wrapper I have this error:
```
┌ Error: problem processing file 
│   run_path = "./run_1.jld2"
│   ex =
│    MethodError: no method matching iterate(::Type{ContactKind})
│    Closest candidates are:
│      iterate(!Matched::Core.SimpleVector) at essentials.jl:603
│      iterate(!Matched::Core.SimpleVector, !Matched::Any) at essentials.jl:603
│      iterate(!Matched::ExponentialBackOff) at error.jl:253
│      ...
└ @ Main ~/Workspace/modelling-ncov2019/julia/src/test.jl:55
```